Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/ErrorHandler.java

```diff
@@ -161,36 +165,40 @@ public ErrorHandler(int errorHorizon, int forecastHorizon, int sequenceIndex, do
     public void update(ForecastDescriptor descriptor, Calibration calibrationMethod) {
         int arrayLength = pastForecasts.length;
         int length = pastForecasts[0].values.length;
-        int errorIndex = sequenceIndex % (arrayLength);
+        int storedForecastIndex = sequenceIndex % (arrayLength);
```

Comment:
yes  -- this code may have to be refactored though In the next set PR/commits we would split the update() step into 
(i) store actuals (ii) calibrate (iii) store the most recent forecasts. The most recent forecasts are not useful for calibration since we have not seen any corresponding actual yet. The update to sequenceIndex has to occur before calibration -- so that calibration is now state dependent and can be invoked from anywhere.